src/scdaq-calo.conf

```diff
@@ -0,0 +1,90 @@
+################################################################################
```

Comment:
Is there a corresponding config file for uGMT? (I can't seem to find it in this commit.. )

Reply:
There isn't, because it is coming from calo branch. However, you are right, it needs to be added. Not sure if already in this PR.

That being said, where can I find "the latest" uGMT config?

Reply:
Hi Petr,

I'll resolve this, because I'm leaving for holidays tomorrow but my feeling is that it should probably go in in this PR. If you prefer doing it in another that's fine too, though.

-d